<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>180</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="chatter.qrc">:/logo_placeholder.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter your username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Enter your password</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="chatter.qrc"/>
 </resources>
 <connections/>
</ui>
